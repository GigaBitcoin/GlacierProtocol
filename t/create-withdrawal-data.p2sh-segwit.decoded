{
  "txid": "ec0f843689af00eb3b0878fac829c20829fe85917608c0cef3f07d270071f77d",
  "hash": "df9769e02e0a66211fe2dce9f396f4f85469474674edf1c689ffab4d777018c0",
  "version": 2,
  "size": 440,
  "vsize": 224,
  "weight": 896,
  "locktime": 0,
  "vin": [
    {
      "txid": "e0e9bb25fb873c4caccdc8ab743c4350310031f2cc077bb90c3f495458860157",
      "vout": 1,
      "scriptSig": {
        "asm": "002065c863c750331cc03efb4daca217e7925cff87f5e9ca9927564234be0d6278e8",
        "hex": "22002065c863c750331cc03efb4daca217e7925cff87f5e9ca9927564234be0d6278e8"
      },
      "txinwitness": [
        "",
        "30440220079f21625291af6a954bbbf32038095907edda102dba3d5b3722c9b37e8d62470220312f83a7d15f7911ed5594582533c26544dc875c0033e39d682e963caa74ce7601",
        "30440220148fad45e78855c831353c682e09c39919af046e6e71b7279489e59964077f4e02206672c3c494773e034d9fe2441ad0f4f21d787ee30ca7245a1736a18922de35a301",
        "522103d14ddcfb6817f5579695bbb3eb3e185887bf2942b031e6f716343b8fe7e9e8e221028fcd46f8614b2cbf318096968242a1e22bcfb6d8f2b6dc939c8c27c347b2937b210315acb550120f4cdcb460d5c49080ab508ca4ccd8dedecac22feeac8cd017d4c121022b063ee2f22f9e1982c140fe778672c683111a796dcccdbd47bb65e3d608a98354ae"
      ],
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 0.06997760,
      "n": 0,
      "scriptPubKey": {
        "asm": "OP_HASH160 ad50fb2489353008e75b0193011ff1f4b76f798c OP_EQUAL",
        "hex": "a914ad50fb2489353008e75b0193011ff1f4b76f798c87",
        "reqSigs": 1,
        "type": "scripthash",
        "addresses": [
          "2N93du8YobdgsHyu3qgBvSyhGUT52utMNeA"
        ]
      }
    },
    {
      "value": 0.03000000,
      "n": 1,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 c3f4800772d58f5429dcb550b1a6a53580175bc1 OP_EQUALVERIFY OP_CHECKSIG",
        "hex": "76a914c3f4800772d58f5429dcb550b1a6a53580175bc188ac",
        "reqSigs": 1,
        "type": "pubkeyhash",
        "addresses": [
          "myP4xdJNwAW9iMakvCjnozg814ewgn8apd"
        ]
      }
    }
  ]
}
