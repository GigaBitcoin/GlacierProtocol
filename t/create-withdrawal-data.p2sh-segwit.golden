Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
You will need to enter several pieces of information to create a withdrawal transaction.


*** PLEASE BE SURE TO ENTER THE CORRECT DESTINATION ADDRESS ***


Source cold storage address: 
Redemption script for source cold storage address: 
Destination address: 
How many unspent transactions will you be using for this withdrawal? 
Please paste raw transaction #1 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Transaction data found for source address.
TOTAL unspent amount for this raw transaction: 0.10000000 BTC

How many private keys will you be signing this transaction with? 
#: Key #1: Key #2: 
Enter fee rate.
Satoshis per byte: 
Based on the provided rate, the fee will be 0.00002250 bitcoin.
Confirm? (y/n): 
Please enter the decimal amount (in bitcoin) to withdraw to the destination address.

Example: For 2.3 bitcoins, enter "2.3".

After a fee of 0.00002250, you have 0.09997750 bitcoins available to withdraw.

*** Technical note for experienced Bitcoin users:  If the withdrawal amount & fee are cumulatively less than the total amount of the unspent transactions, the remainder will be sent back to the same cold storage address as change. ***

Amount to send to myP4xdJNwAW9iMakvCjnozg814ewgn8apd (leave blank to withdraw all funds stored in these unspent transactions): 0.06997750 being returned to cold storage address address 2N93du8YobdgsHyu3qgBvSyhGUT52utMNeA.

Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.10000000 BTC in unspent supplied transactions
0.06997750 BTC going back to cold storage address 2N93du8YobdgsHyu3qgBvSyhGUT52utMNeA
0.03000000 BTC going to destination address myP4xdJNwAW9iMakvCjnozg814ewgn8apd
Fee amount: 0.00002250

Signing with private keys: 
cQCrT9Ncs9729ao7jbmAWrD9z7tF64s2yKzmD6nkiLAi9sXVZWAn
cP65UeSDZPiTLB6CBwasWv9oJYEjRgQXhswfwcT9HscEKDcEbgy4


Confirm? (y/n): 
Calculating transaction...


Sufficient private keys to execute transaction?
True

Raw signed transaction (hex):
020000000001015701865854493f0cb97b07ccf231003150433c74abc8cdac4c3c87fb25bbe9e0010000002322002065c863c750331cc03efb4daca217e7925cff87f5e9ca9927564234be0d6278e8ffffffff02f6c66a000000000017a914ad50fb2489353008e75b0193011ff1f4b76f798c87c0c62d00000000001976a914c3f4800772d58f5429dcb550b1a6a53580175bc188ac040048304502210080a0ad851e5d1337fd448189e83e42f39dd84a9fb93c6f59b83108e5d39cfc71022031643595de324160a1978ea161ac505b2f8248037637796341749d9d6a2c5eef0147304402206954fb982e0226df00389f224fcc6ea8e5f88fddbeb254e99175fc2021f056c002205ccc58dd249f1a9886825239460f466603399967f96c20787c3c8d67e9934f96018b522103d14ddcfb6817f5579695bbb3eb3e185887bf2942b031e6f716343b8fe7e9e8e221028fcd46f8614b2cbf318096968242a1e22bcfb6d8f2b6dc939c8c27c347b2937b210315acb550120f4cdcb460d5c49080ab508ca4ccd8dedecac22feeac8cd017d4c121022b063ee2f22f9e1982c140fe778672c683111a796dcccdbd47bb65e3d608a98354ae00000000

Transaction fingerprint (md5):
f780914649268f7a867082ae0f4ddd85
QR code for transaction written to transaction.png
